test/test_jit.py

```diff
@@ -14961,7 +14965,8 @@ def check(name):
                 args_tensor = deepcopy(unpack_variables(args_variable))
 
                 def fn(*inputs, **kwargs):
-                    output = getattr(inputs[0], name)(*inputs[1:], **kwargs)
+                    attr = getattr(inputs[0], name)
```

Comment:
nit: this isn't necessary, but it's fine to keep it

Reply:
I just broke this up to make it easier to add print statements when I was trying to figure out how to get sparse to work in jit.